src/components/contactUs.css

```diff
@@ -0,0 +1,49 @@
+input[type='text'],
+textarea,
+input[type='email'] {
+  width: 100%;
+  padding-left: 2%;
+  border: none;
+  background-color: #ffffff;
+  border-radius: 10px;
+  resize: none;
+  color: #3f3d56;
+}
+input[type='text'],
+input[type='email'] {
+  height: 82%;
+}
+
+.submitButton {
+  width: 25%;
+  height: 82%;
+  border: 0px;
+  border-radius: 10px;
+  background-color: #ffffff;
+  float: right;
+  text-align: center;
+  color: #3f3d56;
+}
+.textarea {
```

Comment:
Work on responsiveness, text turns to be pretty small in phone display, radius gets too big. Button too tight